src/component/Home.js

```diff
@@ -0,0 +1,72 @@
+import React, {Component} from 'react'
```

Comment:
`Home` and `Favorites` components are similar, you could have abstracted their core logic into another component.